packages/mgt-spfx-utils/README.md

```diff
@@ -0,0 +1,155 @@
+# Utilities for SharePoint Framework Web Parts using Microsoft Graph Toolkit
+
+[![npm](https://img.shields.io/npm/v/@microsoft/mgt-spfx-utils?style=for-the-badge)](https://www.npmjs.com/package/@microsoft/mgt-spfx-utils)
+
+![SPFx 1.15.2](https://img.shields.io/badge/SPFx-1.15.2-green.svg?style=for-the-badge)
+
+Helper functions to simplify lazy loading of Microsoft Graph Toolkit components when using disambiguated web components in SharePoint Framework web parts. For more information on disambiguation in MGT see https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/main/packages/mgt-components#disambiguation
+
+## Installation
+
+To lazy load Microsoft Graph Toolkit components from the library, add the `@microsoft/mgt-spfx-utils` package and if using React, the `@microsoft/mgt-react` package as dependencies to your SharePoint Framework project:
+
+```bash
+npm install @microsoft/mgt-spfx-utils
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils
+```
+
+or when using React:
+
+```bash
+npm install @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+> **Important:** Since a given web component tag can only be registered once these approaches **must** be used along with the `customElementHelper.withDisambiguation('foo')` approach as this allows developers to create disambiguated tag names.
+
+By disambiguating tag names of Microsoft Graph Toolkit components, developers can use their own version of MGT rather than using the centrally deployed `@microsoft/mgt-spfx` package. This allows them to avoid colliding with other SharePoint Framework components, built by other developers. When disambiguating tag names, MGT is included in the SPFx bundle, increasing its size.
```

Comment:
```suggestion
By disambiguating tag names of Microsoft Graph Toolkit components, you can use your own version of MGT rather than using the centrally deployed `@microsoft/mgt-spfx` package. This allows you to avoid colliding with SharePoint Framework components built by other developers. When disambiguating tag names, MGT is included in the generated SPFx bundle, increasing its size.
```